src/main/resources/unsupported_bundles.json

```diff
@@ -0,0 +1,4 @@
+[
```

Comment:
To avoid hardcoding the "unsupported" message in the above code you could use a format like below.

```
{
  "unsupported": [
    {
      "reason": "Unsupported with Java 21",
      "bundles": [
        "org.cid15.aem.groovy.console:aem-groovy-console-bundle:17.0.0",
        "org.objectweb.asm:asm:3.1"
      ]
    }
  ]
}
```

This also allows supporting multiple sets of unsupported bundles (later), by adding similar elements to the `unsupported` array. To allow being YAGNI for now the initial code might only support one element in this array, and loudly complain if there are several elements. But if makes the format future proof and avoids hardcoding the "Java 21" message.